io.github.bgrabitmap.LazPaint.yml

```diff
@@ -0,0 +1,57 @@
+app-id: io.github.bgrabitmap.LazPaint
+runtime: org.kde.Platform
+runtime-version: '5.15-24.08'
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.freepascal
+command: lazpaint
+rename-icon: lazpaint
+rename-desktop-file: lazpaint.desktop
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
```

Comment:
By default it uses this custom image browsing window. It can be switched to default open dialog by disabling menu File - Use image browser. So to keep this original behaviour designed by the authors I should rather add additional permissions like --filesystem=host. I checked other projects like Krita, Blender and others and usually they have it. Which will lead to safety warning but that is how it is designed right now. Other way would be to switch by default to normal open/save dialogs if those have access to the full file system.

Reply:
> ther way would be to switch by default to normal open/save dialogs if those have access to the full file system.

If it works then it's fine to switch to that.

Reply:
Disabled image browser by default and removed filesystem host permission.
Tested after flatpak uninstall --delete-data io.github.bgrabitmap.LazPaint and it can access the files through standard dialogs.